### PR TITLE
basic 'loading...' message and minor search box improvements

### PR DESCRIPTION
**Pull request by its-mahdi**

### CODE REVIEW

File: src/styles/search/_TextAreaSearch.scss [hunk 1]
```diff
@@ -43,19 +43,13 @@
...
 
-    label {
-      width: 50%;
-      display: inline-block;
-      cursor: pointer;
-    }
-
     span {
-      width: 120px;
+      width: 200px;
```
Comment (xwatkins) [inline src/styles/search/_TextAreaSearch.scss]: Could this use another unit than px?

File: src/styles/search/_TextAreaSearch.scss [hunk 2]
```diff
@@ -43,19 +43,13 @@
...
       display: inline-block;
     }
 
     .variant-example {
-      width: 200px;
+      width: 300px;
```
Comment (xwatkins) [inline src/styles/search/_TextAreaSearch.scss]: Could this use another unit than px?

File: src/styles/search/_TextAreaSearch.scss (hunk 1 above)
Comment (its-mahdi) [reply src/styles/search/_TextAreaSearch.scss]: Fixed.

File: src/styles/search/_TextAreaSearch.scss (hunk 2 above)
Comment (its-mahdi) [reply src/styles/search/_TextAreaSearch.scss]: Fixed.

**Pull request merged**